src/contrib/testkits/Akka.TestKit.Xunit2/TestKit.cs

```diff
@@ -50,6 +50,9 @@ public void WriteLine(string format, params object[] args)
         /// </summary>
         protected readonly ITestOutputHelper Output;
 
+        private bool _disposed;
+        private bool _disposing;
```

Comment:
Don't need this

Reply:
Dispose is a blocking long process now, can block for up to 5 seconds, disposing is needed just in case user called dispose multiple times